buildpack.toml

```diff
@@ -60,3 +69,9 @@ api = "0.2"
   [[order.group]]
     id = "paketo-community/puma"
     version = "0.0.5"
+    optional = true
+
+  [[order.group]]
+    id = "paketo-community/unicorn"
+    version = "0.0.1"
+    optional = true
```

Comment:
Rather than making these buildpacks optional, can we have 2 distinct groups as is shown in this RFC: https://github.com/paketo-community/ruby/blob/master/rfcs/0001-web-server-buildpacks.md.

Reply:
Sounds good